pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -334,6 +335,10 @@ def run_evaluation(self, test=False):
             self.checkpoint_callback.on_epoch_end(epoch=self.current_epoch,
                                                   logs=self.callback_metrics)
 
+        # return evaluation results called from Trainer.test()
+        if test:
```

Comment:
why do not return it always?

Reply:
I guess this function could be called for both validation and test evaluations, and in issue #633 it was only required for Trainer.test() to return the evaluation results. However, if called for validation evaluation, it should be logged not returned, as far as I know.

Reply:
then pls return it always :]